aspnetcore/mvc/views/razor.md

```diff
@@ -1,11 +1,11 @@
 ---
-title: Razor Syntax Reference for ASP.NET Core
-author: rick-anderson
-description: Details Razor syntax
-keywords: ASP.NET Core,Razor
+title: Razor syntax reference for ASP.NET Core
+author: ntaylormullen
```

Comment:
You can put yourself + rick-anderson here 😄

Reply:
Sure ... will do ... but u owe me a :beer:. lol :grinning: